SynCommons.pas

```diff
@@ -15879,6 +15879,7 @@   {$ifdef UNICODE}TPrecisionTimer = record{$else}TPrecisionTimer = object{$endif
     // - typical use is to declare a fTimeElapsed: TPrecisionTimer protected
     // member, then call fTimeElapsed.ProfileCurrentMethod at the beginning of
     // all process expecting some timing, then log/save fTimeElapsed.Stop content
+    // FPC TIP: result should be assigned to the local variable of IUnknown type

```

Comment:
typo
// - FPC TIP: ... to a local variable
(add a leading - and use "a" instead of "the")